### ISSUE TITLE
Fill In All Alt Text For Images

### ISSUE THREAD

**Issue by alexfinnarn** [CUCentralAdvancement/essential-cu#418]
All the images should have alt text on them.

**Acceptance Criteria**

- [ ] grep the codebase for `<Image />` and `<img />` tags
- [ ] Make sure all images in CMS have alt text on them
